cmd/querier/main.go

```diff
@@ -91,13 +92,16 @@ func main() {
 	sampleQueryable := querier.NewQueryable(dist, chunkStore, false)
 	metadataQueryable := querier.NewQueryable(dist, chunkStore, true)
 
-	engine := promql.NewEngine(sampleQueryable, nil)
+	maxConcurrent := 20
+	timeout := 2 * time.Minute
+	engine := promql.NewEngine(util.Logger, nil, maxConcurrent, timeout)
 	api := v1.NewAPI(
 		engine,
 		metadataQueryable,
 		querier.DummyTargetRetriever{},
 		querier.DummyAlertmanagerRetriever{},
 		func() config.Config { return config.Config{} },
+		map[string]string{}, // TODO: include configuration flags
```

Comment:
?

Reply:
this is a recent addition to allow flags to be shown via api endpoint - https://github.com/prometheus/prometheus/pull/3864

We will need to include querier flags here.

Reply:
OK; I think we can happily pass blank.